src/toolkits/style_transfer/style_transfer.cpp

```diff
@@ -382,10 +402,10 @@ void style_transfer::perform_predict(gl_sarray data, gl_sframe_writer& result,
                                      const std::vector<double>& style_idx) {
   if (data.size() == 0) return;
 
-  flex_int batch_size = read_state<flex_int>("batch_size");
-  flex_int num_styles = read_state<flex_int>("num_styles");
-  flex_int image_width = read_state<flex_int>("image_width");
-  flex_int image_height = read_state<flex_int>("image_height");
+  int batch_size = read_state<flex_int>("batch_size");
+  int num_styles = read_state<flex_int>("num_styles");
+  int image_width = read_state<flex_int>("image_width");
+  int image_height = read_state<flex_int>("image_height");
```

Comment:
Why are these being changed? Before it was consistent, now it is inconsistent

Reply:
> In the future, please err on the side of not putting unrelated changes into the same PR.

I will take care next time.